contrib/babelfishpg_tsql/src/plan_inval.c

```diff
@@ -80,12 +90,17 @@ static void
 pltsql_initialize_identity_insert_plan(CachedPlanSource *plansource)
 {
 	List *id_insert_info_sublist = NIL;
+	plan_info_enum_list* id_insert_enum;
 	char *id_insert_name;
+
 	tsql_identity_insert_fields *id_insert_state;
 
 	/* Initialize name */
 	id_insert_name = pstrdup(pltsql_identity_insert_name);
```

Comment:
Why do we still need to keep the identity insert name? I think we can safely remove the insert name and make enum as the first entry of the list.